supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -23,8 +23,15 @@ Follow these guidelines when linking externally:
 * Always include link text. Bare URLs (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target.
 * Use hyperlinks unless the URL is an example URL or is otherwise inaccessible to users. For information about using addresses in examples, see the _IBM Style_ guide.
 * By default, links are followable and crawlable. Do not use the "nofollow" link option unless absolutely necessary.
-* Avoid links to external sites when possible. External links can change or be unreliable.
 * Avoid deep links (to a specific page or image from another company) to prevent an ongoing maintenance responsibility and to prevent bypassing a website's legal information.
+* Avoid links to external sites when possible. External links can change or be unreliable.
+External sites are considered to be sites other than those owned and operated by Red Hat.
```

Comment:
```suggestion
External sites are sites other than those owned and operated by Red Hat and IBM.
```

Reply:
Changed to "or IBM". As a subsidiary Red Hat is a separate entity, even though it is wholly owned by IBM.